### ISSUE TITLE
Repeatable attribute flag is considered invalid

### ISSUE THREAD

**Issue by iquito** [vimeo/psalm#4892]
I have the following code in one of my projects:

```php
#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
class DenyAccessUnlessGranted
{
}
```

This leads to the following psalm error (using dev-master 3390097)

```
ERROR: InvalidArgument - DenyAccessUnlessGranted.php:5:14 - Argument 1 of Attribute::__construct expects int(1)|int(10)|int(11)|int(12)|int(13)|int(14)|int(15)|int(16)|int(17)|int(18)|int(19)|int(2)|int(20)|int(21)|int(22)|int(23)|int(24)|int(25)|int(26)|int(27)|int(28)|int(29)|int(3)|int(30)|int(31)|int(32)|int(33)|int(34)|int(35)|int(36)|int(37)|int(38)|int(39)|int(4)|int(40)|int(41)|int(42)|int(43)|int(44)|int(45)|int(46)|int(47)|int(48)|int(49)|int(5)|int(50)|int(51)|int(52)|int(53)|int(54)|int(55)|int(56)|int(57)|int(58)|int(59)|int(6)|int(60)|int(61)|int(62)|int(63)|int(7)|int(8)|int(9), int(69) provided (see https://psalm.dev/004)
#[\Attribute(\Attribute::TARGET_CLASS | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
```

When trying the same code on psalm.dev this error is not reported. When I remove `\Attribute::IS_REPEATABLE` from the attribute definition, then psalm does not report an error. So it seems `\Attribute::IS_REPEATABLE` has a different value on my system than Psalm expects (I am using PHP 8.0, compiled from PHP sources).

**Comment by orklah** [comment]
Somehow, it looks like mask of values have a fixed limit to 64 items, but instead of having a fallback to int, it just stops at 63 and refuse every higher item.
